components/hero/DateSearch.jsx

```diff
@@ -9,21 +9,21 @@ const DateSearch = () => {
   // ]);
   const [dates, setDates] = useState([
     new DateObject().setDay(5),
-    new DateObject().setDay(14).add(1, "month"),
+    new DateObject().setDay(14).add(1, 'month'),

```

Comment:
Why are we hard coding dates?

Reply:
If we have to, always set the dates as 1st April - today for the current financial year